table_pose_tb3p:
  position: [-0.000278813397971, -0.00300640884842, 0.0]
  orientation: [0.0, 0.0, 0.00107402904536, 0.999999423231]

table_pose_one:
  position: [2.41462773979, -0.862754635885, 0.0]
  orientation: [0.0, 0.0, -0.69092172616, 0.722929573555]

table_pose_two:
  position: [2.45515696505, 0.0533631724713, 0.0]
  orientation: [0.0, 0.0, 0.743977935731, 0.668204183723]

table_pose_three:
  position: [0.997289563215, -0.736614710001, 0.0]
  orientation: [0.0, 0.0, -0.721891243913, 0.692006525954]
